docs/manual/contributing/_index.de.md

```diff
@@ -118,13 +118,13 @@ cd contao
 Beim Klonen installierst du das Hugo Learn Theme als Submodul von git.
 
 ```bash
-git clone --recurse-submodules git@github.com:DEIN-GITHUB-BENUTZERNAME/docs.git
+git clone --recurse-submodules git@github.com:DEIN-GITHUB-BENUTZERNAME/docs.git ./
```

Comment:
This will fail if the current directory is not empty.

Reply:
The Manual asks to create a new directory contao...

Reply:
Ah I see what you mean now, because previously we already generated a new folder.

Reply:
Thx, @fritzmg for review and suggestions